### PR TITLE
add more contributing guidance guidance

### LINKED ISSUE #289: governance.md, support.md, roadmap
cc @Bisaloo 

In the chapter about making a package collaboration friendly, mention the possibility to add support.md (link to discuss.ropensci.org in a default one for the org?), governance.md, settings of the repo e.g. making a review compulsory on every PR.

Regarding a roadmap, in practice it might mean using
* GitHub issue labels with a clear indication of where help is needed. Current guidance on labelling https://devguide.ropensci.org/collaboration.html#issue-labelling
* GitHub milestones
* GitHub pinned issues
* GitHub projects (less useful I think for things that are within a single repo?)

### LINKED ISSUE #308: More contributing guide guidance?
cf https://github.com/ropensci/opencage/pull/128#issuecomment-776858806 by @dpprdan 
Linked to #289

### PR DESCRIPTION
**Pull request by maelle**
@Bisaloo would you mind having a look at this? No worries if not. It's my digesting the call.

Fix #308 

Fix #289

### CODE REVIEW

File: maintenance_marketing.Rmd [hunk 1]
```diff
@@ -10,7 +10,7 @@ We will help you promoting your package but here are some more things to keep in
```
Comment (Bisaloo) [inline maintenance_marketing.Rmd:10]: Should we mention `usethis::use_release_issue()` here? It also helps a lot to smooth the submission process and make sure you didn't forget one of the many checks necessary to get it on CRAN

File: maintenance_collaboration.Rmd [hunk 2]
```diff
@@ -28,18 +31,57 @@ And delete the package current code of conduct if there was one.
...
+
+Adding a personal touch to your contributing guide might make it look less generic and therefore more sincere.
+
+CONTRIBUTING.md can also contain some details about how you acknowledge contributions (see [this section](#attributions)) and the roadmap of your package (cf [this example](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md)).
 
 We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/), after making sure you'd see such questions on the forum. Users can use the forum to ask questions about use and report their use cases, and you can subscribe to individual categories and tags to receive notifications about your package.  Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.  Please direct your users to tag posts with the package name.
 
 You can use the `lintr` package's own bot, lintr-bot, via continuous integration, to get comments if a commit or PR deteriorates the code style of your package. See [this link for guidance](https://github.com/jimhester/lintr#continuous-integration).
```
Comment (Bisaloo) [inline maintenance_collaboration.Rmd:56]: Sorry, this is not in the parts you changed but it's somewhat related. I'm not sure I would recommend using a bot that posts comments about style. It creates a lot of noise and extra cognitive load. 

The best solution in my opinion is to use the ['pr-commands' action](https://github.com/r-lib/actions/blob/master/examples/pr-commands.yaml). It prevents premature comments / commits (that would result in conflicts) for people that like to work with many small commits and push before everything is ready.

We can discuss this in another issue if you wish.

File: maintenance_collaboration.Rmd [hunk 3]
```diff
@@ -28,18 +31,57 @@ And delete the package current code of conduct if there was one.
...
+
+* Process preferences? Issue before a PR?
+
+* Scope? E.g. in skimr.
+
+* Sandbox account creation? Mocking in tests? Use links to external docs.
+
+Example: [contributing guide template by Peter Desmet](https://gist.github.com/peterdesmet/e90a1b0dc17af6c12daf6e8b2f044e7c).
+
+Adding a personal touch to your contributing guide might make it look less generic and therefore more sincere.
```
[the remark this answers is not in the document]
Comment (maelle) [reply maintenance_collaboration.Rmd:50]: This came from a remark of yours :grin:

File: maintenance_marketing.Rmd (hunk 1 above)
Comment (maelle) [reply maintenance_marketing.Rmd:10]: It's in https://devdevguide.netlify.app/releasing.html#releasing-1 but maybe I should add a link to that section?

File: maintenance_collaboration.Rmd (hunk 2 above)
Comment (maelle) [reply maintenance_collaboration.Rmd:56]: Good point, I'll rephrase/remove!

**Comment by maelle** [comment]
Merci @Bisaloo!

**Pull request merged**